examples/simple/asyncapi.yaml

```diff
@@ -0,0 +1,50 @@
+asyncapi: 2.4.0
```

Comment:
I told you to update the example, create an actual proper example that would help the users understand how to use glee with http protocol. You can either delete the `simple` example and start new or update it. It is up to you.